src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Set Up Physical Cluster Replication
+summary: Follow a tutorial to set up physical replication between a primary and standby cluster.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} In this tutorial, you will set up physical cluster replication between a primary cluster and standby cluster. The primary is _active_, serving application traffic. The standby is _passive_, accepting updates from the primary cluster. The replication stream will send changes from the primary to the standby.
+
+The unit of replication is a _virtual cluster_, {% comment %}link to Matt's CV docs{% endcomment %} which is part of the underlying infrastructure in the primary and standby clusters.
+
+In this tutorial, you will connect to:
+
+- The _system interface_ for administration tasks in both clusters, and starting the replication stream from the standby cluster.
+- The application _virtual cluster_ on the primary to work with databases, tables, workloads, and so on.
+
+{% include {{ page.version.version }}/physical-replication/reference-links-replication.md %}
+
+## Overview
+
+The high-level steps in this tutorial are:
+
+1. Create and start the primary cluster.
+1. Configure and create a user on the primary cluster.
+1. Create and start the standby cluster.
+1. Configure and create a user on the standby cluster.
+1. Verify certificates.
+1. Start the replication stream from the standby cluster.
+
+## Before you begin
```

Comment:
@mdlinville Perhaps, though this is included in the deployment prerequisite along with the certificate generation that you mention in the next comment. I was trying to avoid duplicating too much content we have in that deployment piece. Having said that, if you think it is hard to follow then I could label where node IPs are with a node number rather than actual IPs.

I followed the deploy tutorial with separate cloud VMs, so maybe that did make a difference.

Reply:
Makes sense. I think when I followed the tutorial, I needed to do it with a new cluster, so I read it that way, but of course others will probably be configuring PCR on an existing cluster that already has certificates.